packages/fx-core/src/plugins/resource/simpleauth/plugin.ts

```diff
@@ -142,8 +142,8 @@ export class SimpleAuthPluginImpl {
     const selectedPlugins = (ctx.projectSettings?.solutionSettings as AzureSolutionSettings)
       .activeResourcePlugins;
     const context = {
-      plugins: selectedPlugins,
-    };
+      Plugins: selectedPlugins,
+    } as BicepPluginsContext;
```

Comment:
the context here should be determined by each resource plugin, instead of asking solution plugin define an interface, because we do not need to limit what context can be used to render bicep files in resource plugin. Any data (e.g. some answers from users) can be used by resource plugin to generate the bicep file, this is also why we ask each resource plugin render their own bicep module files in resource plugin's logic, instead of render them in solution plugin's logic.